travis/test_wordcount.sh

```diff
@@ -1,18 +1,20 @@
 #!/bin/bash
-
-# This script runs WordCount example locally in a few different ways.
```

Comment:
Hi @jbonofre -- it does not look like this text has been re-added in the current version on GitHub. Maybe you did not push a recent commit?